admin_manual/maintenance/upgrade.rst

```diff
@@ -2,93 +2,47 @@
-* Enterprise customers will use their Enterprise software 
-  repositories to maintain their ownCloud servers, rather than the Open Build 
-  Service. Please see :doc:`../enterprise_installation/linux_installation` for 
-  more information.
-
-.. warning:: When upgrading from oC 9.0 to 9.1 with existing Calendars or Adressbooks
-   please have a look at the :doc:`../release_notes` of oC 9.0 for important info
-   about the needed migration steps during that upgrade.
-
-When an update is available for your ownCloud server, you will see a 
-notification at the top of your ownCloud Web interface. When you click the 
-notification it brings you here, to this page.
-
-**It is best to keep your ownCloud server upgraded regularly**, and to install 
-all point releases and major releases without skipping any of them, as skipping 
-releases increases the risk of errors. Major releases are 8.0, 8.1, 8.2, and 
-9.0. Point releases are intermediate releases for each major release. For 
-example, 8.0.9 and 8.1.3 are point releases. **Skipping major releases is not 
-supported.**
-
-**Upgrading is disruptive**. Your ownCloud server will be put into maintenance 
-mode, so your users will be locked out until the upgrade is completed. Large 
-installations may take several hours to complete the upgrade.
-
-.. warning:: **Downgrading is not supported** and risks corrupting your data! If 
-   you want to revert to an older ownCloud version, make a new, fresh 
-   installation and then restore your data from backup. Before doing this, 
-   file a support ticket (if you have paid support) or ask for help in the 
-   ownCloud forums to see if your issue can be resolved without downgrading.
-
-.. not sure about notifications
-.. Update Notifier and Updater App Are Not the Same
-.. ------------------------------------------------
-
-.. ownCloud has two update tools: the ownCloud core update notifier, and the 
-.. Updater app. Figure 1 shows what you see when the Updater app is enabled: 
-.. both 
-.. the core notifier and the Updater app control panel are visible on your 
-.. admin 
-.. page.
-
-.. .. figure:: images/2-updates.png
-..   :alt: Both update mechanisms displayed on Admin page.
-   
-..   *Figure 1: The top yellow banner is the update notifier, and the Updates 
-..   section is the Updater app.*
-   
-.. The core update notifier has only one function, and that is to display a 
-.. notification when a new ownCloud release is available. Then you decide which 
-.. upgrade method to use. When you maintain your ownCloud server via your Linux 
-.. package manager you should ensure that the Updater app is disabled.
+Before beginning an upgrade, please keep the following points in mind:
+
+- Review :doc:`the release notes </release_notes>` for important information about the needed migration steps during that upgrade to help ensure a smooth upgrade process.
+- Skipping major releases and downgrading **is not supported**.
```

Comment:
Perhaps this can stay written like this for this PR. However it needs to be updated some time to document that from a certain point direct upgrades to 10.*.* are supported (need to double-check exactly what that is - I don't want to guess out of my head).
So this would need to become 2 points - one about the major release upgrade support, and one saying that downgrading is not supported at all.